packages/larva-patterns/components/c-label/c-label.twig

```diff
@@ -0,0 +1,19 @@
+{% if c_label_tag == 'h2' %}
```

Comment:
Is this syntax supported in the parser? Make sure to test these Twig updates in the pmc-plugins/pmc-larva/_core instance of larva to make sure everything integrates okay - there should be a corresponding PR to that repo when any Twig files are updated.